.github/workflows/docs.yml

```diff
@@ -24,9 +24,11 @@ jobs:
           node-version: ${{ matrix.node }}
       - name: "Git Fetch & Checkout"
         run: |
+          git config --local user.email "team@fluvio.io"
+          git config --local user.name "Fluvio Authors"
           git fetch --all
-          git branch -D docs
           git checkout -b docs --track origin/docs
+          git merge origin/master
```

Comment:
merge or rebase?

Reply:
merge should be used here. `docs` branch is never rebased back into master. We only want to merge in the upstream changes in master. No need to rebase here.